src/main/java/fr/insalyon/creatis/gasw/plugin/executor/dirac/execution/DiracMonitor.java

```diff
@@ -306,10 +310,18 @@ public synchronized void add(String jobID, String symbolicName, String fileName,
 
     @Override
     protected void kill(Job job) {
+        this.kill(job, false);
+    }
+
+    protected void killRunnnigJob(Job job) {
```

Comment:
This is probably silly, but I'm not sure to understand why you need the killRunningJob method.